src/index.md

```diff
@@ -1461,7 +1464,7 @@ After the initial release of the JDK source code into OpenJDK in 2007 the OpenJD
 
 This document assumes familiarity with the first two chapters of the free on-line book [Mercurial: The Definitive Guide](http://hgbook.red-bean.com).
 
-## Installing and Configuring Mercurial
+## Installing and configuring mercurial
```

Comment:
I think you may want to capitalize "Mercurial" as that's the name of a tool.  They seem to follow this convention in their documentation.  Regardless of what you decide, I think the case on this line should match with the one on new line 1505 "Cloning a Mercurial repository" and other similar subtitles.

Reply:
Yes, Mercurial should have a capital M. Good catch!